canonical-web-and-design/snapcraft.io
# Duplicate fsf routes twice

## Done
- _Add all the routes for `/first-snap-2` and `/first-snap-3`._
- _Make login required for all the pages in first-snap-2 and first-snap-3._

## How to QA
- Visit the demo in your browser, link commented below
  - Or you can run the project locally using the [dotrun](https://snapcraft.io/dotrun) snap with `$ dotrun` and view it in your web browser at: http://0.0.0.0:8004/first-snap
  - If your demo doesn't work, read more about the [demoservice](https://discourse.ubuntu.com/t/demoservice/16862)
- Run through the following [QA steps](https://canonical-web-and-design.github.io/practices/workflow/qa-steps.html)
- Run through the FSF and replace the `first-snap` in the url with `first-snap-2` or `first-snap-3` on any page and see it still works, but only if you are logged in

## Issue / Card
Fixes https://github.com/canonical-web-and-design/snap-squad/issues/1761

## Screenshots
[if relevant, include a screenshot]
